src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -234,7 +248,7 @@ private void preAbortCheck() {
     /**
      * Check if the current user can submit the input.
      */
-    private void preSubmissionCheck() {
+    public void preSubmissionCheck() {
```

Comment:
Merge with `master` as this was already done in https://github.com/jenkinsci/pipeline-input-step-plugin/commit/e8990e57b1654cfdb2fccc3fe96b8a6d08994c4f; is this PR still necessary?